public/data/viroopaksh-utrayak.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "viroopaksh-utrayak",
+  "type": "personal",
+  "bio": "well I'm a programmer(java) but first of all I'm a learner",
+  "avatar": "https://github.com/viroopaksh-utrayak.png",
+  "links": [
+    {
+      "name": "Here is my Github Accnt",
+      "url": "https://github.com/viroopaksh-utrayak",
+      "icon": "github"
+    },
+    {
+      "name": "Here is my Twitter Accnt",
+      "url": "https://twitter.com/utrayak",
+      "icon": "twitter"
+    },
+    {
+      "name": "Here is my Instagram Accnt",
+      "url": "https://www.instagram.com/viroopaksh_utrayak/",
+      "icon": "instagram"
+    },
+    {
+      "name": "Here is my  LinkedIn Accnt",
```

You also can do lower-case all these accounts if you want. 
```suggestion
      "name": "Here is my LinkedIn Account",
```